packages/sanity/src/core/form/studio/tree-editing/utils/useLegacyArrayEditingConfig.ts

```diff
@@ -0,0 +1,13 @@
+import {useSource} from 'sanity'
+
+/**
+ * A utility hook to check if the array editing dialog should be open (based on config)
+ * @returns Returns true if the dialog should be open
+ * @internal
+ */
+export function useLegacyArrayEditingConfig(): boolean {
```

Comment:
Given that the name of this hook implies it should return the _config_, should we return the entire config (i.e, `document.unstable_legacyArrayEditing `) instead of just the `enabled` property from the config? If we decide to return only the `enabled` property (as we do currently) – should we consider renaming the hook to `useLegacyArrayEditingEnabled()` to better reflect its purpose?

Reply:
Done 👍